helios/datatypes/__init__.py

```diff
@@ -186,7 +186,14 @@ def serialize(self):
     
     def toDict(self, alternate_fields=None, complete=False):
         val = {}
-        for f in (alternate_fields or self.FIELDS):
+
+        fields = alternate_fields or self.FIELDS
```

Comment:
@shirlei do you want to fix this PR as per the comment above and then I can merge it?